src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -0,0 +1,148 @@
+import {Box, Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+
+import {t} from '../../../locale';
+import ApiMixin from '../../../mixins/apiMixin';
+import AsyncView from '../../asyncView';
+import Confirm from '../../../components/confirm';
+import EmptyMessage from '../components/emptyMessage';
+import IndicatorStore from '../../../stores/indicatorStore';
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import Row from '../components/row';
+import SettingsPageHeader from '../components/settingsPageHeader';
+
+const TeamRow = createReactClass({
+  displayName: 'TeamRow',
+
+  propTypes: {
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    team: PropTypes.object.isRequired,
+    access: PropTypes.object.isRequired,
+    onRemove: PropTypes.func.isRequired,
+  },
+
+  mixins: [ApiMixin],
+
+  getInitialState() {
+    return {
+      loading: false,
+      error: false,
+    };
+  },
+
+  handleRemove() {
+    if (this.state.loading) return;
+
+    let loadingIndicator = IndicatorStore.add(t('Saving changes..'));
+    let {orgId, projectId, team} = this.props;
+    this.api.request(`/projects/${orgId}/${projectId}/teams/${team.slug}/`, {
+      method: 'DELETE',
+      success: (d, _, jqXHR) => {
+        this.props.onRemove();
+        IndicatorStore.remove(loadingIndicator);
+      },
+      error: () => {
+        this.setState({
+          error: true,
+          loading: false,
+        });
+        IndicatorStore.remove(loadingIndicator);
+      },
+    });
+  },
+
+  render() {
+    let team = this.props.team;
+    return (
+      <Row justify="space-between" px={2} py={2}>
```

Comment:
you can use `p` for `px` + `py`, also is `justify: space-between` needed? the name has flex=1, so will take up all available space and we should similar behavior.

Reply:
err i think this was mostly copypasta from some other view 😅  so probably not. i'll try removing